server/src/controller/card/card.controller.ts

```diff
@@ -24,11 +23,10 @@ export class CardController {
         const card = request.body;
 
         try {
-            const isSave = await this.cardService.saveCard(card.id, card.name, card.surname, card.image, card.skills, card.hp, card.damage);            
-            if (isSave) {                
-                response.status(200).send({status: 'Saved'});
-            } else {
-                response.status(400).send({status: 'Error'});
+            await this.cardRepository.saveCard(card.id, card.name, card.surname, card.image, card.skills, card.hp, card.damage);
+
+            if (response.statusCode == 200) {    
```

Comment:
??? you never set response status code, please remove unnecessary "if", just return response with corresponding status